packages/atomic/src/components/result-template-components/atomic-result-printable-uri/atomic-result-printable-uri.pcss

```diff
@@ -13,3 +13,7 @@ atomic-result-printable-uri li::after {
 atomic-result-printable-uri li:last-child::after {
   content: none;
 }
+
+atomic-result-printable-uri atomic-result-text {
```

Comment:
saw a quick improvement there, the long urls that printable URI displays sometimes overflows